stack88.yaml

```diff
@@ -3,6 +3,7 @@ packages:
 - .
 extra-deps:
 - fuzzy-0.1.0.0
+- hie-bios-0.4.0
```

Comment:
Note this will clash with my changes in #400 - but I'm happy to rebase after this lands.